packages/cli/lib/lib/babel-config.js

```diff
@@ -3,7 +3,7 @@ module.exports = function (env, options = {}) {
 
```

Comment:
Someone who really knows what they're doing probably needs to take a deep look at this. I think the general idea is to remove non-syntax stuff for `optimize-plugin`, but I'm not confident I didn't totally bugger up someone's app.